test/components/remote_retro_test.js

```diff
@@ -32,4 +39,28 @@ describe("RemoteRetro component", () => {
       expect(hotjarSpy).calledWith("trigger", CLOSED)
     })
   })
+
+  describe("when component updates with a new facilitator", () => {
+    let newFacilitator
+    beforeEach(() => {
+      newFacilitator = {
+        ...stubUser,
+        is_facilitator: true,
+      }
+    })
+
+    it("calls the new facilitator action", () => {
+      const wrapper = shallow(<RemoteRetro {...defaultProps} />)
+      wrapper.setProps({ user: newFacilitator })
+      expect(defaultProps.actions.newFacilitator.calledWith(defaultProps.retro))
+    })
+  })
+
+  describe("when component updates new stage", () => {
+    it("does not the new facilitator action", () => {
```

Comment:
based on the implementation code below, i think the test setup here is a little misleading, as the condition checks whether there is specifically a change in the currentUser's `is_facilitator` boolean attribute (from *not* being facilitator *to* being facilitator). So in the `setProps()` call, we would make the component update by including a currentUser who is still *not* the facilitator